raspberry/bzzz/read_sbus/read_sbus_from_GPIO_receiver.py

```diff
@@ -112,7 +116,14 @@ def get_radio_data_parse_and_send_to_ESP(self,
                                              force_send_fake_data=False,
                                              fake_data="",
                                              over_write_throttle_ref_to=-1):
-        """Read the radio data, process it, format it into a string, and send it via UART.
+        """
+        Read the radio data, process it, format it into a string, and send it via UART.
+
+        :param return_channel_data: 
```

Comment:
Sorry if this is a stupid question, but why are we using the syntax `:param param_name:` inside comment blocks?

Reply:
In Python docstrings, the syntax 

```py
"""
:param paramName: description
"""
```

is used to write documentation about function parameters. See [this tutorial](https://www.jetbrains.com/help/pycharm/using-docstrings-to-specify-types.html#example). 

I noticed that in [`CONTRIBUTING.md`](https://github.com/QUB-ASL/bzzz/blob/main/CONTRIBUTING.md#33-documentation-and-comments) we state that we should be using `@param`, but we're using `:param:` throughout the project as far as I can tell. 

We should update [`CONTRIBUTING.md`](https://github.com/QUB-ASL/bzzz/blob/main/CONTRIBUTING.md#33-documentation-and-comments). @ejb-11 can you take care of this (create a new branch out of `dev` and then create a PR).

Reply:
Thanks for explaining.

Reply:
@ejb-11 In CONTRIBUTING.md, Section 3.3 is about documentation in general (both Python and C++). Actually, `@param` is for C++. For Python we've written that 

> In Python, the code should be documented using [Sphinx](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) docstrings.

Perhaps this is enough, since there is a link, but if you feel it would be better to have more detailed instructions there, feel free to propose an update.